posthog/api/organization.py

```diff
@@ -176,6 +177,8 @@ def get_object(self):
     def perform_destroy(self, organization: Organization):
         user = cast(User, self.request.user)
         report_organization_deleted(user, organization)
+        team_ids = [team.pk for team in organization.teams.all()]
+        delete_clickhouse_data.delay(team_ids=team_ids)
```

Comment:
non-blocking

I guess there's a failure case where we rollback postgres, but have deleted clickhouse data. Probably not a case that we care about too much but thought I'd point out.